### PR TITLE
Feat: Move last popup utils to global utils dir

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Review by pstachel — APPROVED** [review]

File: src/utils/i18nTranslations.ts [hunk 1]
```diff
@@ -34,28 +34,40 @@ export const TX_TYPE_TRANSLATIONS: TranslationMap<TxType> = {
...
   NamePreclaimTx: () => tg('transaction.listType.namePreclaimTx'),
   NameClaimTx: () => tg('transaction.listType.nameClaimTx'),
   NameUpdateTx: () => tg('transaction.listType.nameUpdateTx'),
   NameTransferTx: () => tg('transaction.listType.nameTransferTx'),
   NameRevokeTx: () => tg('transaction.listType.nameRevokeTx'),
   SpendTx: () => tg('transaction.listType.sentTx'),
 };
+export function getTxTypeListLabel(txType: TxType): string {
+  const translateFunc = TX_TYPE_LIST_TRANSLATIONS[txType];
+  return translateFunc ? translateFunc() : '';
```
Comment (konradodo) [inline src/utils/i18nTranslations.ts:56]:
```suggestion
  return TX_TYPE_LIST_TRANSLATIONS[txType] ? translateFunc() : '';
```

File: src/utils/i18nTranslations.ts [hunk 2]
```diff
@@ -34,28 +34,40 @@ export const TX_TYPE_TRANSLATIONS: TranslationMap<TxType> = {
   PayingForTx: () => tg('transaction.type.payingForTx'),
   SpendTx: () => tg('transaction.type.sentTx'),
 };
+export function getTxTypeLabel(txType: TxType): string {
+  const translateFunc = TX_TYPE_TRANSLATIONS[txType];
+  return translateFunc ? translateFunc() : '';
```
Comment (konradodo) [inline src/utils/i18nTranslations.ts:39]:
```suggestion
  return TX_TYPE_TRANSLATIONS[txType] ? translateFunc() : '';
  
```

File: src/utils/i18nTranslations.ts [hunk 3]
```diff
@@ -34,28 +34,40 @@ export const TX_TYPE_TRANSLATIONS: TranslationMap<TxType> = {
...
   propose: () => tg('transaction.function.propose'),
   revoke: () => tg('transaction.function.revoke'),
   refuse: () => tg('transaction.function.refuse'),
   confirm: () => tg('transaction.function.confirm'),
   tip_token: () => tg('transaction.function.tip_token'),
   retip_token: () => tg('transaction.function.retip_token'),
 };
+export function getTxFunctionLabel(txFunction: TxFunctionRaw | TxFunctionMultisig): string {
+  const translateFunc = TX_FUNCTION_TRANSLATIONS[txFunction];
+  return translateFunc ? translateFunc() : '';
```
Comment (konradodo) [inline src/utils/i18nTranslations.ts:72]:
```suggestion
  return TX_FUNCTION_TRANSLATIONS[txFunction] ? translateFunc() : '';
```

**Review by konradodo — APPROVED** [review]

File: src/utils/i18nTranslations.ts (hunk 2 above)
Comment (peronczyk) [reply src/utils/i18nTranslations.ts:39]: This change makes no sense as you first suggest to remove the `translateFunc` constant and than you use it.

File: src/utils/i18nTranslations.ts (hunk 1 above)
Comment (peronczyk) [reply src/utils/i18nTranslations.ts:56]: This change makes no sense as you first suggest to remove the `translateFunc` constant and than you use it.

File: src/utils/i18nTranslations.ts (hunk 3 above)
Comment (peronczyk) [reply src/utils/i18nTranslations.ts:72]: This change makes no sense as you first suggest to remove the `translateFunc` constant and than you use it.

**Pull request merged**